WooCommerce/WooCommerceTests/ViewRelated/Products/Actions Factory/ProductFormActionsFactoryTests.swift

```diff
@@ -741,6 +741,59 @@ final class ProductFormActionsFactoryTests: XCTestCase {
         XCTAssertEqual(factory.bottomSheetActions(), expectedBottomSheetActions)
     }
 
+    func test_view_model_for_subscription_product_when_downloadable() {
```

Comment:
np: maybe `test_view_model_for_subscription_product_when_is_downloadable()` flows better as a sentence?

same with the next one: test_view_model_for_subscription_product_when_is_not_downloadable()`

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/11231/commits/f15934cdb408510add5802429dc4056c3e35170a